package.json

```diff
@@ -3,14 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
     "react-router-dom": "^6.4.5",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
+    "redux-logger": "^3.0.6",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
![Capture0](https://user-images.githubusercontent.com/93039359/206868746-48846388-6a8a-4fdb-80ac-4da62de213b2.PNG)

- [ ] You have a link of your license file in your readme file 😊 but it is broken 😨 Kindly fix this link to make your readme file descriptive and license file easily accessible to all viewers 👍